### ISSUE TITLE
Thrift  autogen

### ISSUE THREAD

**Issue by vasilcovsky** [concord/concord-go#2]
Please publish thrift files. Concord-go should use **go generate** to generate Thrift client/server

**Comment by emaxerrno** [comment]
yes! we should. let's chat about how to generate files for the different runtimes. 

as of today we have 2 runtimes. so give me a bit to think through what to publish. need to trim down the boring fat of these files

**Comment by vasilcovsky** [comment]
any progress on this?

**Comment by emaxerrno** [comment]
Everything is ready to go... the build is updated too. Will ask the team on Wednesday to see if we can release. Thanks for bringing this up Igor! /cc @shinjikim @RBlafford
